packages/ember-extension-support/lib/data_adapter.js

```diff
@@ -93,6 +93,20 @@ export default EmberObject.extend({
   attributeLimit: 3,
 
   /**
+   * A change in Ember and Ember Data
+   * required string model names to be passed
+   * around instead of the actual factories.
```

Comment:
Can you note the version? beta.18. It will make it easier to know when this should be stripped.